Gem/Code/Source/Components/Multiplayer/WeaponEffectComponent.h

```diff
@@ -18,13 +18,7 @@ namespace MultiplayerSample
     public:
         AZ_MULTIPLAYER_COMPONENT(MultiplayerSample::WeaponEffectComponent, s_weaponEffectComponentConcreteUuid, MultiplayerSample::WeaponEffectComponentBase);
 
-        static void Reflect(AZ::ReflectContext* context);
-
-        static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
-        {
-            WeaponEffectComponentBase::GetRequiredServices(required);
-            required.push_back(AZ_CRC("PopcornFXEmitterService"));
```

Comment:
would it make more sense to wrap this in one of those new CLIENT #defines so it is required on the clients?

Reply:
Those don't apply at reflection time.